packages/scaffolding-cli/templates/src/csr/public/manifest.json

```diff
@@ -1,25 +1,44 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "name": "Amido Stacks",
```

Comment:
When I build this locally, I still see the title as `React App`. Might be worth changing?

Reply:
The title of the application on the browser is driven from the index.html and not from manifest. I guess the manifest dictates the PWA or mobile app icon etc. I will fix the title soon.

Reply:
**Sorted now. Try it now**